osu.Game/Tests/Visual/OsuTestScene.cs

```diff
@@ -317,6 +317,15 @@ public static APIBeatmapSet CreateAPIBeatmapSet(IBeatmapInfo original)
             return result;
         }
 
+        public static APIRelation CreateAPIRelationFromAPIUser(APIUser user) =>
+            new APIRelation
+            {
+                Mutual = true,
+                RelationType = RelationType.Friend,
+                TargetID = user.OnlineID,
+                TargetUser = user
+            };
```

Comment:
Why is this in `OsuTestScene`? Should not be. Put it in a static utils method or something. Or even inline it in the few tests that use it.